api/proto/config/acipherinfo.proto

```diff
@@ -70,4 +70,8 @@ message EncryptionBlock {
   string wifiUserName = 3;      // If the authentication type is EAP
   string wifiPassword = 4;
   string protectedUserData = 5;
+  // Username for cellular network.
+  string cellNetUsername = 6;
```

Comment:
How about using lower_snake_case for these two new fields.

Reply:
Done